src/drivers/mailbox/Mybuild

```diff
@@ -2,6 +2,9 @@ package embox.driver.mailbox
 
 module bcm2835_mailbox {
 	@IncludeExport(path="drivers/mailbox")
+
+	option number base_addr = 0x2000B880
+
```

Comment:
I would suggest to declare this option before IncludeExport, because IncludeExport annotaion concerns mailbox.h header. So it should be:

```cpp
module bcm2835_mailbox {
	 option number base_addr = 0x2000B880

         @IncludeExport(path="drivers/mailbox")
	 source "bcm2835_mailbox.h"

	 source "bcm2835_mailbox.c"
}
```